bin/test

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env -S npx bats
+
+@test "The bundled OAI document should contain Legacy Doc link fragment:" {
```

Comment:
(nit) Maybe a short comment that this is just a sanity check that the decorator works?